test/NuGet.Core.Tests/NuGet.ProjectModel.Test/DependencyGraphSpecTests.cs

```diff
@@ -214,18 +163,18 @@ public void Load_WithPath_WhenJsonIsInvalidDgSpec_Throws(string json)
             }
         }
 
-        [Theory]
-        [InlineData("{}{}")]
-        [InlineData("{}[]")]
-        public void Load_WithPath_WhenJsonContainsMultipleTopLevelEntities_IgnoresNonFirstEntities(string json)
-        {
-            using (Test test = Test.Create(json))
-            {
-                DependencyGraphSpec dgSpec = DependencyGraphSpec.Load(test.FilePath);
+        //[Theory]
+        //[InlineData("{}{}")]
+        //[InlineData("{}[]")]
+        //public void Load_WithPath_WhenJsonContainsMultipleTopLevelEntities_IgnoresNonFirstEntities(string json)
+        //{
+        //    using (Test test = Test.Create(json))
+        //    {
+        //        DependencyGraphSpec dgSpec = DependencyGraphSpec.Load(test.FilePath);
 
-                Assert.Equal("{}", dgSpec.Json.ToString());
-            }
-        }
+        //        Assert.Equal("{}", dgSpec.Json.ToString());
+        //    }
+        //}
```

Comment:
I'd delete this test, rather than comment it out.

It's a rather strange edge case. Also, I don't understand why this behaviour would be desirable, let alone prevented from changing. If the file is invalid (for example it contains more than one top-level json token), why would we even want DGSpec.Load to succeed?

I think it would be better to treat it as a missing DGSpec, and fail the no-op check, thereby doing a full restore. Anyway, this PR isn't about fixing weird tests, but I think deleting this test is fine. Leaving it in the code, but commented out is an unusual choice. Why not add a `Skipped` reason to the `Theory` attribute?

Reply:
Meant to remove it. 

That test is asserting a behavior that's not relevant anymore